src/edit.js

```diff
@@ -1,19 +1,42 @@
-import countries from '../assets/countries.json';
-import './editor.scss';
-import { edit, globe } from '@wordpress/icons';
-import { BlockControls } from '@wordpress/block-editor';
-import { ComboboxControl, Placeholder, ToolbarButton, ToolbarGroup } from '@wordpress/components';
-import { getEmojiFlag } from './utils';
+/**
+ * WordPress dependencies
+ */
+import { edit as editIcon, globe as globeIcon } from '@wordpress/icons';
+
+import { BlockControls, useBlockProps } from '@wordpress/block-editor';
+
+import {
+	ComboboxControl,
+	Placeholder,
+	ToolbarButton,
+	ToolbarGroup,
+} from '@wordpress/components';
+
+import apiFetch from '@wordpress/api-fetch';
+
 import { useEffect, useState } from '@wordpress/element';
+
 import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import countries from '../assets/countries.json';
+import { getEmojiFlag } from './utils';
+
 import Preview from './preview';
 
-export default function Edit( { attributes, setAttributes } ) {
+export default function Edit( { attributes, setAttributes, context } ) {
+	const blockProps = useBlockProps();
+
+	const { postId } = context;
+
 	const { countryCode, relatedPosts } = attributes;
-	const options = Object.keys(countries).map(code => ({
-        value: code,
-        label:  getEmojiFlag( code ) + '  ' + countries[code] + ' — ' + code
-	}));
+	const options = Object.keys( countries ).map( ( code ) => ( {
+		value: code,
+		label: getEmojiFlag( code ) + '  ' + countries[ code ] + ' — ' + code,
+	} ) );
```

Comment:
@cdevroe Do you think the options array calculation could be improved in some way?